### ISSUE TITLE
ijk我印象里是用于流播放的，能用来做边下边播么

### ISSUE THREAD

**Issue by codeliu6572** [Bilibili/ijkplayer#2479]
ijk我印象里是用于流播放的，能用来做边下边播么？像迅雷看看那样有缓存，倒回去看的时候播放的是缓存的，不耗流量，希望得到回复

**Comment by xinzhengzhang** [comment]
#2375

**Comment by codeliu6572** [comment]
谢谢，通过#2375，我了解到是因为缓存协议不支持在移动设备，你说会做一个补丁来制定一个文件进行存储，不知道我理解的对不？这项技术目前ijk是不支持的吧？

**Comment by xinzhengzhang** [comment]
Yes, you have to specify the cache file path and it is not supported for now.

**Comment by codeliu6572** [comment]
Thank you for your reply, thank you very much! Best wishes!

**Comment by OCer** [comment]
@codeliu6572 你解决了吗  我用的0.7.7版  cache:rtmp://live.hkstv.hk.lxdns.com/live/hks
根本就播放不了

**Comment by codeliu6572** [comment]
@OCer 这么目前没有好的解决方案，下载需要版权，缓存目前不知道支不支持，现在没有再关注了。

**Comment by jepack** [comment]
https://juejin.im/entry/5a2f51ca6fb9a044fa19d327 看下这个 支持ij k
